db/migrate/20170801091621_add_foreign_key_to_product_nature_variant.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyToProductNatureVariant < ActiveRecord::Migration
+  def change
+    add_reference :product_nature_variants, :valuing, index: true
```

Comment:
A foreign key constraint must be added.
If possible, can you merge the 2 migrations in order to be consistent and update timestamp of migration in order to be last one after those of master branch?